pages/index.html

```diff
@@ -4,9 +4,37 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>Michelle's Portfolio Website</title>
+  <link href="/styles/style.css" rel="stylesheet">
+  <script src="https://kit.fontawesome.com/3e8fd03e4e.js"
+  crossorigin="anonymous"></script>
 </head>
+
 <body>
-  
+<content id="main">
```

Comment:
[content](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/content) is no longer a valid html tag, and was not a general-use tag. It was used as part of a web components standard, but has since been deprecated and should not be used.

Running your html through a validator would catch this.